docs/package.json

```diff
@@ -5,9 +5,9 @@
     "d3-request": "^1.0.5",
     "immutable": "^4.0.0-rc.1",
     "markdown-loader": "^2.0.0",
-    "react": "^15.4.2",
-    "react-addons-shallow-compare": "^15.4.2",
-    "react-dom": "^15.4.2",
+    "react": "^15.5.4",
+    "react-addons-shallow-compare": "^15.5.2",
```

Comment:
Looks like this is deprecated. Will add an issue for task to refactor into using `React.PureComponent`.